components/automate-ui/src/app/pages/project/rules/project-rules.component.scss

```diff
@@ -16,9 +16,8 @@ chef-page {
     overflow-y: scroll;
   }
 
-  chef-input {
+  input {
     height: 45px;
```

Comment:
@susanev - This seems to be the only place where 45px needs to be explicitly set because it is surrounded by selects that are set to 45px.  Not sure if this has been seen, would be helpful if we could make all the inputs the same uniform height....or maybe they are already supposed to be?

Reply:
yes, really want all the inputs the same size.
im good with whatever height makes the most sense for everything
heres an example where the difference is kinda bad
![create-team-modal](https://user-images.githubusercontent.com/5489125/68332653-0656e180-008c-11ea-9aab-e65ba00735e9.png)